README.md

```diff
@@ -28,6 +30,18 @@ To run a test suite:
 npm run test
 ```
 
+To run a specific test file:
+
+```sh
+npx playwright test <name-of-test-file>
+```
+
+To run tests in headed state:
+
+```sh
+npm run test-h || npx playwright test <name-of-test-file> --headed
```

Comment:
Please remove references to `npx`. Usage of `npx` for the task is advised against as you are not 100% using the version specified in package.json. Please add scripts for the tasks. 

You can use `--` to pass parameters to scripts. Please provide an example and document.

Reply:
done